### ISSUE TITLE
[LicenceDocument] complicated way to name a Standard Licence?

### ISSUE THREAD

**Issue by peterlubrich** [mobilityDCAT-AP/mobilityDCAT-AP#55]
Very often, we want to name a Standard Licence for a dataset, e.g. "CC BY-NC".

For this purpose, we use the property [licence](https://mobilitydcat-ap.github.io/mobilityDCAT-AP/releases/index.html#recommended-properties-for-distribution) for class Distribution.

This property is meant to "...refer to the licence under which the Distribution is made available. This SHOULD be a reference to a concrete standard license."

However, we found a very complicated way to name a standard licence under class Licence Document [here](https://mobilitydcat-ap.github.io/mobilityDCAT-AP/releases/index.html#properties-for-licence-document):

In the definition for class Licence Document, the standard licence is named under a (new) property "[standard licence](https://mobilitydcat-ap.github.io/mobilityDCAT-AP/releases/index.html#licence-document-standard-licence)". 

However, it would be easier to just name the standard licence as the range of property [licence](https://mobilitydcat-ap.github.io/mobilityDCAT-AP/releases/index.html#recommended-properties-for-distribution) for class Distribution.

Example:
```
<dcat:distribution>
   <dct:license rdf:resource="http://publications.europa.eu/resource/authority/licence/CC_BY_4_0"/>
<dcat:distribution>
```

Easy and conforms to DCAT-AP! 
So, let's give up the property "[standard licence](https://mobilitydcat-ap.github.io/mobilityDCAT-AP/releases/index.html#licence-document-standard-licence)"!

**Comment by marioscrock** [comment]
I agree with suggesting the usage of this "short" representation for well-known licenses, and we can also recommend the usage of this vocabulary (http://publications.europa.eu/resource/authority/licence) that I think is currently not mentioned in the spec. However, I would keep also the "verbose" possibility not to break backward compatibility. I think it was added to allow the specification of "custom licenses" since we have an optional property of LicenseDocument to add the license text as a string.

**Comment by marioscrock** [comment]
Related also to #27

**Comment by hildea** [comment]
I also agree that we should conform to DCAT-AP for licence.

I see that GeoDCAT-AP has removed the analogue property [licence text](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#licence-document-licence-text) in their 3.0.0 version.

Regarding the suggestion from @marioscrock to keep the verbose possibility for backwards compatibility: Do we know of any cases where NAPs use this field for custom licences today? If there are such examples, we can examine those to find out if we really need a special property just for mobilityDCAT-AP to describe them.

**Comment by kimm-hoang** [comment]
BE NAP: We use the property licence text.

**Comment by marioscrock** [comment]
I agree that we should try to conform to DCAT-AP as much as possible. My understanding is that also in DCAT-AP, the `Distribution` is related with `dct:license` to a `LicenseDocument` that should have a property `dct:type` that refers to a value in a recommended vocabulary (BTW they recommend the [ADMS](http://purl.org/adms/licencetype/1.0) one instead of the [Authority Licence](http://publications.europa.eu/resource/authority/licence) which I believe is better). 

However, in my experience and if you look at datasets on data.europa.eu (e.g., this [one](https://data.europa.eu/api/hub/repo/datasets/18489cb7-bce7-4d44-a138-795b390d2109~~1.ttl?useNormalizedId=true&locale=en)), the property `dct:license` is also directly used in DCAT-AP descriptors with a URI value representing the license (as proposed by @peterlubrich) and with values from the [Authority Licence](http://publications.europa.eu/resource/authority/licence) vocabulary. 

I believe both cases can be admissible, similar to what we discussed in #65 for `mobilityDataStandard`. However, if we decide to remove the property [licence text](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#licence-document-licence-text), it probably does not make sense to have the "more verbose" one.

To summarise, I make two proposals:

Proposal 1 (keep both options)
- [ ] clarify usage of `dct:license` to directly reference a licence URI or an instance of `LicenceDocument`. In both cases recommend the usage of the [Authority Licence](http://publications.europa.eu/resource/authority/licence) vocabulary
- [ ] adapt SHACL shapes to accept both behaviour
- [ ] remove the label "standard licence" but use simply "type" for the property of LicenceDocument as done in [DCAT-AP](https://semiceu.github.io/DCAT-AP/releases/3.0.0/#LicenceDocument)

Proposal 2 (simplify)
- [ ] remove `LicenceDocument` properties (including [licence text](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#licence-document-licence-text)) and specify that `dct:license` should directly reference a licence URI, possibly from the [Authority Licence](http://publications.europa.eu/resource/authority/licence) vocabulary
- [ ] adapt SHACL shapes accordingly

Thanks @hildea for pointing out the changes made by GeoDCAT-AP regarding the property [licence text](https://semiceu.github.io/GeoDCAT-AP/releases/2.0.0/#licence-document-licence-text). I believe we inherited the property from GeoDCAT-AP, so we may consider in the discussion the related issue in their repository https://github.com/SEMICeu/GeoDCAT-AP/issues/113.

**Comment by barthanssens** [comment]
For "known" licenses, the "short" form,  a URI from the EU authority table, should indeed be used.

(To be clear, it would still be allowed to specify a ADMS license type in dct:type - see https://github.com/mobilityDCAT-AP/mobilityDCAT-AP/issues/135#issuecomment-3486036495 - on the LicenseDocument, but mostly for keeping SHACL validators happy.
In the perfect world, this info should be available on the web, e.g on the portal of the EU Publication office... just don't use the type to reference a LicenseDocument)


I wouldn't recommend setting the text / label on known licenses, because in practice this could lead to slightly different labels - so a series of multiple labels  - when combining different sources, but no strong opinion about it. 

There will always be custom licenses, so it would be good practice IMHO to set the dct:type on these, and definitely to include a license text / label (in the perfect world the data publisher could define a persistent URI and publish the text, type etc as linked data, but that won't be always the case)

**Comment by peterlubrich** [comment]
There is some clear advice in this discussion that would streamline the way to tell a (standard) licence. 

This advice is the same as given by geoDCAT-AP colleagues in this [issue](https://github.com/SEMICeu/GeoDCAT-AP/issues/152). 

Thus: I will formulate a concrete proposal in #165 and close this issue.
